delivery_purchase_label/models/delivery_carrier.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2024 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import fields, models
+
+
+class DeliveryCarrier(models.Model):
+    _inherit = "delivery.carrier"
+
+    purchase_label_picking_type = fields.Many2one(
+        string="Operation Type For Purchase Label", comodel_name="stock.picking.type"
+    )
```

Comment:
Is this really needed to be set on the carrier?
Or is it enough to just simple configure the carrier on the partner?
And the for creating the purchase delivery label - just use the operation type from data xml?

This would result in less code and it would ensure that there is just one operation type for this matter. 

```suggestion
```

Reply:
yes the feature needs be enabled, but it is already enabled on the partner where the carrier is set.

Reply:
The background to this question is.
How do you find out which transfers are "purchase deliver label" transfers. 

The simplest way would be to have just one operation type for all of those transfers. This would mean no configuration on carrier just on the supplier. 

With the current implementation there could be a lot types used for that. 
Which makes it a lot harder to find the right transfers. 

A other possible way would be to mark those operation types as delivery purchase label type.